api/pkg/apis/v1alpha1/managers/solution/solution-manager.go

```diff
@@ -543,7 +543,13 @@ func (s *SolutionManager) Reconcile(ctx context.Context, deployment model.Deploy
 			for _, v := range targetResult {
 				successCount += v
 			}
-			summary.CurrentDeployed += successCount
+			deployedCount := 0
+			for _, ret := range componentResults {
+				if ret.Status == v1alpha2.Updated { // TODO: need to ensure the status updated correctly on returning from target providers.
```

Comment:
Btw, if it is delete case, will the expected return status is also Updated?

Reply:
it should return "Deleted" but some providers are still returning "Untouched". I updated the status check here and will add this to issue #513